source/isaaclab/isaaclab/sensors/tacsl_sensor/gelsight_render.py

```diff
@@ -0,0 +1,234 @@
+# Copyright (c) 2022-2025, The Isaac Lab Project Developers (https://github.com/isaac-sim/IsaacLab/blob/main/CONTRIBUTORS.md).
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import numpy as np
+import os
+import scipy
+import torch
+
+import cv2
+
+from isaaclab.utils.assets import ISAACLAB_NUCLEUS_DIR, retrieve_file_path
+
+conf_r15 = {
+    "data_dir": "gelsight_r15_data",
+    "background_path": "bg.jpg",
+    "calib_path": "polycalib.npz",
+    "real_bg": "real_bg.npy",
+    "h": 320,
+    "w": 240,
+    "numBins": 120,
+    "pixmm": 0.0877,
+}
+conf_gs_mini = {
+    "data_dir": "gs_mini_data",
+    "background_path": "bg.jpg",
+    "calib_path": "polycalib.npz",
+    "real_bg": "real_bg.npy",
+    "h": 240,
+    "w": 320,
+    "numBins": 120,
+    "pixmm": 0.065,
+}
```

Comment:
Following all other sensors, please make these into configclass objects with the correct doc-strings. We usually don't want to keep sensor-specific settings in the core codebase.

Sensors configuration live here: https://github.com/isaac-sim/IsaacLab/tree/main/source/isaaclab_assets/isaaclab_assets/sensors